sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,22 +1,18 @@
 # Release History
 
-## 1.2.1 (Unreleased)
+## 1.2.1 (2021-07-26)
```

Comment:
Shouldnt this be 1.3.0 because of the core-tracing changes we made?

@maorleger,
Maybe it is worth doing a sweep of all the packages that use core-tracing and ensure that they are lined up for a minor version update along with a changelog entry?

Reply:
There's also the core-http 2.0 and targeting es2017 which is also part of this change - I do not see changelog entries for it, maybe we need to add them? What we've been using is:

```
- Support for Node.js 8 and IE 11 has been dropped. Please see our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
- Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
- Updated our internal core package dependencies to their latest versions in order to add support for Opentelemetry 1.0.0 which is compatible with the latest versions of our other client libraries.
```

Reply:
I think updating minor versions makes sense, it lines up with what we've done in storage and will do in key vault.

Reply:
Updated.. does that look good?